### PR TITLE
Tooltip changes for 10.0.2

### PR DESCRIPTION
**Pull request by Silarn**
Tooltip interfaces were changed in 10.0.2, causing errors.

This has fixed all of the issues I've discovered so far.

### CODE REVIEW

**Comment by Silarn** [comment]
I actually don't recommend merging this at the moment, there seems to be some sort of performance / conflict happening with the tooltips. I'm trying to verify that it's being caused by MogIt.

**Comment by Silarn** [comment]
Actually this could be ArkInventory or even another AddOn.

**Comment by Silarn** [comment]
I'm pretty sure Ark is having problems, so this fix may be fine.

**Comment by Lombra** [comment]
Hi,

I have some tooltip fixes that I apparently didn't get around to pushing. I'll check this out later.

File: Core/Tooltip.lua [hunk 1]
```diff
@@ -8,7 +8,7 @@ local class = L.classBits[select(2, UnitClass("PLAYER"))];
 
 
 --// Tooltip
-mog.tooltip = CreateFrame("Frame", "MogItTooltip", UIParent, "TooltipBorderedFrameTemplate");
+mog.tooltip = CreateFrame("GameTooltip", "MogItTooltip", UIParent, "GameTooltipTemplate");
```
Comment (Lombra) [inline Core/Tooltip.lua:11]: Is this actually necessary? Despite the naming this frame is not actually a tooltip.

**Pull request merged**